acos_client/v30/base.py

```diff
@@ -59,6 +60,8 @@ def _request(self, method, action, params, retry_count=0, **kwargs):
                 return self._request(method, action, params, retry_count+1,
                                      **kwargs)
             raise e
+#        except json.JSONDecodeError as jsonex:
```

Comment:
Was this added for debugging and should be removed, or is it something you want to implement later?

Reply:
Should be removed. I'll followup with a patch to delete this comment.